service-tests/cocoapods.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const {
+    isMetric,
+    isMetricOverTimePeriod,
+    isIntegerPercentage,
+    isVPlusDottedVersionAtLeastOne,
+} = require('./helpers/validators');
+
+const isPlatform = Joi.string().regex(/^\w+( \| \w+)*$/);
```

Comment:
This is a fairly broad regex. We could call it `isPipeSeparatedWords`, though perhaps it would be better to lock it down: e.g. `/^[osx|ios|tvos|watchos]( \| [osx|ios|tvos|watchos])*$/`.